helm-charts/FATE/templates/pulsar.yaml

```diff
@@ -134,7 +134,7 @@ data:
     brokerDeleteInactiveTopicsEnabled=true
     
     # How often to check for inactive topics
-    brokerDeleteInactiveTopicsFrequencySeconds=60
+    brokerDeleteInactiveTopicsFrequencySeconds=36000
```

Comment:
My understanding is, 60 is too frequent so will eat too much resource, is that right?